cookiecutter.json

```diff
@@ -10,5 +10,6 @@
     "email_password": "",
     "sentry_dsn": "",
     "app_name": "things",
-    "model_name": "Thing"
+    "model_name": "Thing",
+    "model_name_lower": "{{ cookiecutter.model_name|lower }}"
```

Comment:
I expect putting `model_name_lower` here means it will get prompted for when run interactively. If that's the case I'd say we remove it, or put it in a hook.